sdk/Lusid.Sdk.Tests/tutorials/Ibor/Properties.cs

```diff
@@ -148,5 +150,74 @@ public void Create_Portfolio_With_Metric_Property()
             Assert.That(metricProperty.Value.MetricValue.Value, Is.EqualTo(metricPropertyValueRequest.MetricValue.Value));
             Assert.That(metricProperty.Value.MetricValue.Unit, Is.EqualTo(metricPropertyValueRequest.MetricValue.Unit));
         }
+        
+        [Test]
+        public void Create_Portfolio_With_MultiValue_Property()
+        {
+            var uuid = Guid.NewGuid().ToString();
+            var propertyName = $"manager-{uuid}";
```

Comment:
Should this be manager**s** as it is a multi-value property?

Reply:
Yes, changed.